src/plugins/discover/public/application/components/doc/use_es_doc_search.ts

```diff
@@ -69,12 +68,18 @@ export function useEsDocSearch({
         const indexPatternEntity = await indexPatternService.get(indexPatternId);
         setIndexPattern(indexPatternEntity);
 
-        const { hits } = await esClient.search({
-          index,
-          body: buildSearchBody(id, indexPatternEntity),
-        });
+        const { rawResponse } = await getServices()
+          .data.search.search({
+            params: {
+              index,
+              body: buildSearchBody(id, indexPatternEntity),
+            },
+          })
+          .toPromise();
```

Comment:
@lizozom @lukasolson Could you tell me if that's the correct usage of the new APIs, or if I am missing anything here?

Reply:
Seems all right!